.circleci/config.yml

```diff
@@ -59,8 +59,6 @@ workflows:
       # manual approval to run e2e tests.
       - hold:
           type: approval
-          requires:
```

Comment:
This was preventing us from approving the e2e tests execution until `push-azure-operator-to-control-plane-app-catalog` finished. Now they can be approved from the beginning of the workflow.